cylc/flow/cylc_subproc.py

```diff
@@ -33,7 +33,8 @@ def procopen(cmd, bufsize=0, executable=None, stdin=None, stdout=None,
              stderr=None, preexec_fn=None, close_fds=False, usesh=False,
              cwd=None, env=None, universal_newlines=False, startupinfo=None,
              creationflags=0, splitcmd=False, stdoutpipe=False,
-             stdoutout=False, stderrpipe=False, stderrout=False):
+             stdoutout=False, stderrpipe=False, stderrout=False,
+             stdindevnull=False):
```

Comment:
In our usages, STDIN should probably default to `/dev/null` for safety. There are only very small number of cases where we actually want to pipe the *current* STDIN to the child command.

Reply:
Got it, changed :+1: